kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterFactoryContext.java

```diff
@@ -31,7 +31,12 @@ public interface FilterFactoryContext {
      * @param instanceName The plugin instance name
      * @return The plugin instance
      * @param <P> The plugin manager type
-     * @throws UnknownPluginInstanceException
+     * @throws UnknownPluginInstanceException can be thrown when the requested plugin could not be loaded
      */
     <P> @NonNull P pluginInstance(@NonNull Class<P> pluginClass, @NonNull String instanceName);
+
+    @NonNull
+    default String virtualClusterName() {
```

Comment:
the method is called `getVirtualClusterName` on `FilterContext`, I think we should use the same name.

Reply:
I felt that `virtualClusterName` was the more consistent with the FilterFactoryContext interface.